array-manipulation-1/src/initial.ts

```diff
@@ -0,0 +1,11 @@
+/* exported initial */
+function initial(array: any): any {
+  if (array.length <= 1) {
+    return [];
+  }
+  const answer: unknown[] = [];
+  for (let i = 0; i < array.length - 1; i++) {
```

Comment:
You wouldn't need this check here because your `for` loop has a condition statement. If that condition evaluates to `false` then we would not run the code in the loop and skip past it, which would then just go to return your `answer` variable, which is an empty array.